gallery/src/styles/NxCard/NxCardLayoutExample.tsx

```diff
@@ -0,0 +1,296 @@
+              100
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer"><a href="#" className="nx-text-link">Link</a></footer>
+        </section>
+        <section className="nx-card" aria-label="Card with NxReadOnly">
+          <div className="nx-card__content">
+            <dl className="nx-read-only">
+              <dt className="nx-read-only__label">
+                Component Foo
+              </dt>
+              <dd className="nx-read-only__data">
+                Component Foo does not contain proprietary packages
+              </dd>
+              <dt className="nx-read-only__label">
+                Component Bar
+              </dt>
+              <dd className="nx-read-only__data">
+                Component Bar does not contain proprietary packages
+              </dd>
+            </dl>
+          </div>
+        </section>
+        <section className="nx-card" aria-label="Icon in callout">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Icon in callout</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+      </div>
+      <div className="nx-card-container">
+        <section className="nx-card" aria-label="Another card with icon in callout">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">Data point details</div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+        <section className="nx-card" aria-label="card 2 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-status-indicator">
+              <span>Server one</span>
+            </div>
+            <div className="nx-status-indicator">
+              <span>Server two</span>
+            </div>
+          </div>
+          <footer className="nx-card__footer">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+        <section className="nx-card" aria-label="card 2 in row">
```

Comment:
Many of the `aria-label`s from here down no longer make sense

Reply:
> Many of the `aria-label`s from here down no longer make sense

Cleaned up here: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/7db7311c2ebc66b585cbd3c79f11add0ec040e84

Reply:
> Cleaned up here: 7db7311

That's an improvement but the numbers assigned to each card ("card 7" etc) still don't seem to make sense; they neither match the cards index in its container nor its index overall

Reply:
> > Cleaned up here: [7db7311](https://github.com/sonatype/sonatype-react-shared-components/commit/7db7311c2ebc66b585cbd3c79f11add0ec040e84)
> 
> That's an improvement but the numbers assigned to each card ("card 7" etc) still don't seem to make sense; they neither match the cards index in its container nor its index overall

Everything has a better `aria-label` now: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/6ec8ec9daf8d17faea5f865409f9ec997663e2bd